src/predicates.c

```diff
@@ -1105,7 +1105,7 @@ REAL *e;
 /*                                                                           */
 /*****************************************************************************/
 
-REAL orient2dfast(const REAL * pa, const REAL * pb, const REAL * pc)
+predicates_EXPORT REAL orient2dfast(const REAL * pa, const REAL * pb, const REAL * pc)
```

Comment:
The export declaration is [only necessary in header files](https://stackoverflow.com/questions/557317/is-declspecdllexport-needed-in-cpp-files).